README.md

```diff
@@ -411,6 +412,15 @@ Example 2 (using Compound to combine both types):
     Measurements: 'Running*'
 ```
 
+#### Run script (Introduced in QTM 2024.3)
+This analysis runs a script using the [QTM Scripting Interface](https://qualisys.github.io/qtm-scripting/) and supports execution on multiple measurements.
+Each measurement is opened separately, the script is executed and then the measurement is closed.
+The script is selected using the `Filename` property.
+Measurements are selected using `Measurements` and `Exclude` properties.
+- **Filename**: Required. The path and name of the input file. It uses the ``Templates`` directory of the project as a base when the path is relative.
+- **Measurements**: Optional. Selected measurements. See: [Overview of analyses](#overview-of-analyses)
+- **Exclude**: Optional. See [Overview of analyses](#overview-of-analyses)
+
```

Comment:
Please add an example. It would be very useful

Reply:
Added an example